sample_metadata/parser/sample_file_map_parser.py

```diff
@@ -1,20 +1,30 @@
+#!/usr/bin/env python3
 # pylint: disable=too-many-instance-attributes,too-many-locals,unused-argument,no-self-use,wrong-import-order,unused-argument
 from typing import List
 import logging
 
-from sample_metadata.parser.generic_metadata_parser import GenericMetadataParser
+import click
 
-SAMPLE_ID_COL_NAME = 'Individual ID'
+from sample_metadata.parser.generic_metadata_parser import (
+    GenericMetadataParser,
+    run_as_sync,
+)
+
+PARTICIPANT_COL_NAME = 'Individual ID'
+SAMPLE_ID_COL_NAME = 'Sample ID'
 READS_COL_NAME = 'Filenames'
+SEQ_TYPE_COL_NAME = 'Type'
 
 __DOC = """
 The SampleFileMapParser is used for parsing files with format:
 
-- 'Individual ID'
+- ['Individual ID']
+- 'Sample ID'
 - 'Filenames'
+- ['Type']
 
 EG:
-    Individual ID   Filenames
+    Sample ID   Filenames
```

Comment:
Do you mind adding a second example here with the Participant Id and Sequence type?

Reply:
Done!